lib/osf-components/addon/components/tos-consent-banner/component.ts

```diff
@@ -67,6 +67,6 @@ export default class TosConsentBanner extends Component {
         this.analytics.click('button', 'ToS Consent Banner - dismiss');
         this.set('didValidate', false);
         this.currentUser.set('showTosConsentBanner', false);
-        return false;
+        return true;
```

Comment:
Why do we need a return value here in the first place?

Reply:
It is used by the [dismiss action in the component](https://github.com/CenterForOpenScience/ember-osf-web/pull/1861/files#diff-6a4fc2e1b69d5d089dd2e2e180b8bbf37c547d146d0d4f7594d076df74ef0030R41).